ocaml/database/db_xml.ml

```diff
@@ -118,15 +119,22 @@ module From = struct
 						let ctime = match ctime_l with | [(_,ctime_s)] -> Int64.of_string ctime_s | _ -> 0L in
 						let mtime = match mtime_l with | [(_,mtime_s)] -> Int64.of_string mtime_s | _ -> 0L in
 						let row = List.fold_left (fun row ((_, k), v) -> 
-                                                        let table_schema = Schema.Database.find tblname schema.Schema.database in
-                                                        let column_schema = try
-                                                                Schema.Table.find k table_schema
-                                                        with Not_found ->
-                                                                raise (Unmarshall_error (Printf.sprintf "Unexpected column in table %s: %s" tblname k))
-                                                        in
-                                                        let value = Schema.Value.unmarshal column_schema.Schema.Column.ty (Xml_spaces.unprotect v) in
-                                                        let empty = column_schema.Schema.Column.empty in
-							Row.update mtime k empty (fun _ -> value) (Row.add ctime k value row)
+							let table_schema = Schema.Database.find tblname schema.Schema.database in
+							try
+								let column_schema = Schema.Table.find k table_schema in
+								let value = Schema.Value.unmarshal column_schema.Schema.Column.ty (Xml_spaces.unprotect v) in
+								let empty = column_schema.Schema.Column.empty in
+								Row.update mtime k empty (fun _ -> value) (Row.add ctime k value row)
+							with Not_found ->
+								match tblname, k with
+								| "VM_guest_metrics", "network_paths_optimized"
```

Comment:
Do we really need this? You said in a previous commit that it was safe to take these fields out because they weren't in a released product. I realise they are in TP but we don't support upgrade from TP so we don't need this magic?